subsys/tracing/sysview/Kconfig

```diff
@@ -32,4 +32,11 @@ config SEGGER_SYSVIEW_SECTION_DTCM
 
 endchoice
 
+config SEGGER_SYSVIEW_CUSTOM_ISR_NAMES
```

Comment:
please add to new kconfig option to 4.4. release notes

Reply:
Is this comment addressed to me?

If so, how/where do I do that?